k8s/namespaces/family-public-law/fpl-case-service/fpl-case-service.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  labels:
+    hmcts.github.com/envInjector: enabled
+    slackChannel: fpla-tech
+  name: fpl-case-service
+  annotations:
```

Comment:
Dont need thes ehere, they are added by platform by default

Reply:
can you please let me know which ones are added by platform?